docs/build.rst

```diff
@@ -0,0 +1,122 @@
+..
+   Author: Adam Turner
+
+
+Building PEPs Locally
+=====================
+
+Whilst editing a PEP, it is useful to review the rendered output locally. This
+can also be used to check that the PEP is valid reStructuredText before
+submission to the PEP editors.
+
+The rest of this document assumes you are working from a local clone of the
+`PEPs repository <https://github.com/python/peps>`__, with Python 3.9 or later
+installed.
+
+
+Render PEPs locally
+-------------------
+
+1. Install requirements
+
+   .. code-block:: console
+
+      $ python -m venv venv
+      $ . venv/bin/activate
+      (venv) $ python -m pip install --upgrade pip
```

Comment:
Unless I'm mistaken `wheel` is only required when attempting to install a package that doesn't have wheels in the index, and even then setuptools can install packages without it.

I haven't checked if every hard dependency on Sphinx has wheels uploaded, but I have never conciously installed it and I've never had any problems.

A
A

Reply:
Ah, I mostly work in conda envs and only really use venvs on constrained platforms, so I wasn't actually 100% sure; I'd just recalled getting warnings about `wheel` not being available and pip falling back to sdists instead after creating a fresh venv, which I misinterpreted as being that pip couldn't _install_ wheels rather than simply not being able to _build_ wheels from the sdist on platforms that don't have them available. If in PEP 517 mode, pipe _should_ install `wheel` in the isolated build env, assuming its declared in `build_requires` (or the default is used). In any case, this isn't really a priority.

Reply:
Ah, I mostly work in conda envs and only really use venvs on constrained platforms, so I wasn't actually 100% sure; I'd just recalled getting warnings about `wheel` not being available and pip falling back to sdists instead after creating a fresh venv, which I misinterpreted as being that pip couldn't _install_ wheels rather than simply not being able to _build_ wheels from the sdist on platforms that don't have them available. If in PEP 517 mode, pipe _should_ install `wheel` in the isolated build env, assuming its declared in `build_requires` (or the default is used). In any case, this isn't really a priority.

Reply:
Yep. I also don't want to be responsible for providing excuses to retain legacy build systems ;-)

A